swarm/api/api.go

```diff
@@ -343,6 +343,13 @@ func (self *Api) Get(manifestKey storage.Key, path string) (reader *storage.Lazy
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 			rsrc, err := self.resource.LookupLatestByName(ctx, entry.Hash, true, &storage.ResourceLookupParams{})
+			if err != nil {
+				apiGetNotFound.Inc(1)
+				status = http.StatusNotFound
+				log.Warn(fmt.Sprintf("get resource content error: %v", err))
```

Comment:
Is this an expected condition / bad request / not found? I can see we are returning 404 as a status, if this is the case probably `log.Debug` is more appropriate than `Warn`.